pkg/adapter/awssqssource/adapter.go

```diff
@@ -138,8 +152,10 @@ func (a *adapter) queueLookup(queueName string) (*sqs.GetQueueUrlOutput, error)
 // occurs that error will be returned.
 func (a *adapter) getMessages(queueURL string) ([]*sqs.Message, error) {
 	params := sqs.ReceiveMessageInput{
-		AttributeNames: aws.StringSlice([]string{sqs.QueueAttributeNameAll}),
-		QueueUrl:       &queueURL,
+		AttributeNames:      aws.StringSlice([]string{sqs.QueueAttributeNameAll}),
+		QueueUrl:            &queueURL,
+		MaxNumberOfMessages: aws.Int64(maxReceiveMsgBatchSize),
```

Comment:
Improvement n.1: allow receiving more than 1 message at once. SQS caps this number at 10.